### PR TITLE
chore: reorg comms, prepare for new versions

### PR DESCRIPTION
**Pull request by menduz**
Pending: 
- add https://github.com/decentraland/kernel/pull/112/files
- add https://github.com/decentraland/kernel/pull/112/files
-

### CODE REVIEW

File: packages/shared/comms/context.ts [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+  }
+}
+
+let currentParcelTopics = ''
+let previousTopics = ''
+
+let lastNetworkUpdatePosition = new Date().getTime()
+let lastPositionSent: Position | undefined
+
+function onPositionUpdate(context: CommsContext, p: Position) {
```
Comment (pablitar) [inline packages/shared/comms/context.ts:361]: Should this be in context.ts? Or should this be in handlers.ts?

File: packages/shared/comms/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,253 @@
+import { genericAvatarSnapshots, COMMS_PROFILE_TIMEOUT } from 'config'
```
Comment (pablitar) [inline packages/shared/comms/handlers.ts:1]: I'm assuming this code was moved without changes. If there is a change, please let me know!

File: packages/shared/comms/context.ts [hunk 3]
```diff
@@ -0,0 +1,445 @@
+import { commConfigurations, parcelLimits } from 'config'
```
Comment (pablitar) [inline packages/shared/comms/context.ts:1]: I'm assuming most of this code was moved without changes, other than converting some global functions into methods of comms context.

If there's something more significant than that that I should review, please let me know!

File: packages/shared/dao/types.ts [hunk 4]
```diff
@@ -33,19 +32,14 @@ type BaseCandidate = {
...
-
 export type IslandsBasedCandidate = {
   type: 'islands-based'
   usersCount: number
   usersParcels?: Parcel[]
   maxUsers?: number
 } & BaseCandidate
 
-export type Candidate = LayerBasedCandidate | IslandsBasedCandidate
+export type Candidate = IslandsBasedCandidate
```
Comment (pablitar) [inline packages/shared/dao/types.ts:42]: Maybe we can remove the type `IslandBasedCandidate`?

File: packages/shared/comms/index.ts [hunk 5]
```diff
@@ -896,45 +106,60 @@ async function getPeerParameters(): Promise<PeerParameters> {
...
     }
 
+    const user = await getStoredSession(identity?.address)
+
+    if (!user) {
+      return
+    }
+    
     const userInfo = {
       userId,
```
Comment (pablitar) [inline packages/shared/comms/index.ts:146]:
```suggestion
      userId: identity.address,
```

File: packages/shared/dao/types.ts (hunk 4 above)
Comment (menduz) [reply packages/shared/dao/types.ts:42]: 👍

**Pull request closed without merging**